src/pybind/mgr/dashboard/module.py

```diff
@@ -25,6 +26,12 @@
 
 from .services.sso import load_sso_db
 
+ip_addrs = subprocess.check_output(["ip","addr"])
```

Comment:
I vote to use a different solution, e.g. by using procfs.

Reply:
hi, @votdev what's the advantage of procfs?

Reply:
I'm agreeing with Volker here that calling an external program is not my preferred solution.

Reply:
Also this is the wrong place for this if statement, because it's part of the dashboard configuration. Instead it should be located somewhere is `_configure()`

Reply:
A Python "universal" solution would be preferable here.
Otherwise part of it will degrade into a `if (os_platform == .....) ... else .... " version